src/ai-did-you-mean-this/azext_ai_did_you_mean_this/_style.py

```diff
@@ -0,0 +1,26 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import sys
+
+import colorama  # pylint: disable=import-error
+
+
+def style_message(msg):
+    if should_enable_styling():
+        try:
+            msg = colorama.Style.BRIGHT + msg + colorama.Style.RESET_ALL
+        except KeyError:
+            pass
+    return msg
+
+
+def should_enable_styling():
+    try:
+        if sys.stdout.isatty():
+            return True
+    except AttributeError:
+        pass
+    return False
```

Comment:
Please also honor the no_color setting: https://github.com/Azure/azure-cli/pull/12601